static/js/publisher/release/components/progressiveBar.js

```diff
@@ -54,18 +68,32 @@ class InteractiveProgressiveBar extends React.Component {
 
     this.onWheelHandler = this.onWheelHandler.bind(this);
 
-    window.addEventListener("mouseup", this.onMouseUpHandler);
-    window.addEventListener("mousemove", this.onMouseMoveHandler);
+    // If the element isn't disabled, add event liteners
+    if (!props.disabled) {
```

Comment:
Should they also be disabled when readonly?

Reply:
For the `InteractiveProgressiveBar` I figured the name suggests it's not readonly, so I removed the prop (it just passes `readonly={false}` to `ProgressiveBar`. I guess I forgot to remove the `readonly` checks from render.